### ISSUE TITLE
Fix crash on Samsung devices after picture is taken

### ISSUE THREAD

**Issue by cpetzel** [onfido/onfido-android-sdk#34]
#### What was the expected behaviour?
App doesn't crash
#### What happened instead?
App crashes after picture is taken

See more info here
http://crashes.to/s/631620df70d

```
Fatal Exception: java.lang.NullPointerException: Attempt to invoke virtual method 'float android.graphics.RectF.width()' on a null object reference
       at com.onfido.android.sdk.capture.ui.camera.CaptureActivity.onPictureCaptured(Unknown Source)
       at com.onfido.android.sdk.capture.ui.camera.face.CameraSourcePreview$2.onPictureTaken(Unknown Source)
       at com.onfido.android.sdk.capture.ui.camera.face.CameraSource$g.onPictureTaken(Unknown Source)
       at android.hardware.Camera$EventHandler.handleMessage(Camera.java:1115)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:154)
       at android.app.ActivityThread.main(ActivityThread.java:6823)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1563)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1451)
```

#### Version info:

- `onfido-capture-sdk` version: 4.3.0
- Android API version(s):

- `compileSdkVersion`: 27
- `targetSdkVersion`: 27
- `minSdkVersion`: 18
- Support library version (if applicable):

#### Steps to reproduce:
Take picture on samsung device

#### Issue severity information (number of users/crashes if applicable):
misbehaving samsung devices...
SM-G950U : 50%
SM-N950F : 33%
SM-A530F : 17%

#### Other useful info:
_screenshots, videos, etc._

**Comment by sergiomarquesmoura** [comment]
Hi @cpetzel . Thanks for flagging this, we will try to replicate it with a similar device and we will get back to you ASAP

**Comment by sergiomarquesmoura** [comment]
Hi @cpetzel . We're trying to replicate this on a Samsung Galaxy S9 running Android 8.0. However, we couldn't yet. In case you have been able to reproduce it, could you provide us the steps?

**Comment by cpetzel** [comment]
A lot of our users are not on the latest and greatest hardware (S9) and Android version. This crash is happening on a few specific hardware devices. We would expect your SDK to work on all versions of android devices. 

I would suggest that you get your hands on this specific Samsung device, such as [this one](https://www.amazon.com/Samsung-Unlocked-Certified-Refurbished-Midnight/dp/B0764J52SG and fix the errors. 

Thanks!

**Comment by sergiomarquesmoura** [comment]
Hi @cpetzel ! This issue was fixed on version 4.5.0. You can check it using the regular ways.
Again, thanks for flagging
